software.md

```diff
@@ -0,0 +1,31 @@
+# FOSS United Community Software Policy
+This policy is devised to guide, but not mandate, software choices within the 
+FOSS United Community.
+
+- Define what we (the FOSS United Community) wish to do. Ask ourselves 
+whether what we wish to do will help in full-fulling the community's goals.
```

Comment:
Not clear again. Could you please explain this with an example?

Reply:
If I use the BBB's example itself. The intended purpose was to hold monthly community meetups. We ask ourselves whether holding monthly meetups would contribute towards fulfilling the goals of FOSS United. If yes we do it or we don't.

Organizing anything requires work (higher the scale and complexity, the more the work) on top of that this whole process of determining needs, assessing various options, setting up services etc. would obviously require a lot of effort. If what we intend to do doesn't help our goals, should we really do it? Should we waste our time debating about things which in the end doesn't help us? I don't see a reason why we should and that's what this line is about.